javascript/phone-number/phone-number.js

```diff
@@ -0,0 +1,39 @@
+var PhoneNumber = function(input){
+	//validate the input type
+	if(typeof input !== "string") {
+		throw "Invalid input type. Please give a number in string and try again.";
```

Comment:
Nice! Always a good idea to have checks in place to handle input error. It's probably better to throw a `TypeError` rather than a simple `String` though.